docs/content/concepts/partitions-schedules-sensors/asset-sensors.mdx

```diff
@@ -297,7 +297,7 @@ If we materialize partition `"a"` of `asset2`, each upstream dependency of parti
   This feature is currently <strong>experimental</strong>.
 </Note>
 
-Setting a `FreshnessPolicy` on an asset specifies how up-to-date the data in that asset is expected to be. The asset reconciliation sensor created using `build_asset_reconciliation_sensor` will kick off runs to keep that data in line with the defined policy.
+Setting a <PyObject object="FreshnessPolicy"/> on an asset specifies how up-to-date the data in that asset is expected to be. The asset reconciliation sensor created using `build_asset_reconciliation_sensor` will kick off runs to keep that data in line with the defined policy.
```

Comment:
I think also worth linking to the api docs for `build_asset_reconciliation_sensor`